packages/metal-dom/src/domNamed.js

```diff
@@ -296,6 +297,28 @@ export function exitDocument(node) {
 }
 
 /**
+ * Search by a custom event already registered and returns its configuration object.
+ * @param {!string} eventName The custom event name used to find the configuration
+ * object as an object key or by executing a regular expression.
+ * @return {Object} The custom event configuration object or undefined if the given
+ * eventName doesn't match with any custom event name.
+ */
+function getCustomEventData(eventName) {
+	var customConfig = customEvents[eventName];
+	if (!customConfig) {
+		for (var i = customUnamedEvents.length; i--; ) {
```

Comment:
Any specific reason to go through these events in reverse order? I think it'd make more sense to loop them in the order that they were registered, so that the precedence is clear in that sense.